### PR TITLE
Add support for Asus ROG Strix B550-I Gaming motherboard

### PR DESCRIPTION
**Pull request by Pomianowski**

### CODE REVIEW

File: LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs [hunk 1]
```diff
@@ -2365,6 +2365,91 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
...
+                            v.Add(new Voltage("VTT", 9));
+                            //v.Add(new Voltage("#Unused #9", 9, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #10", 10, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #11", 11, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #12", 12, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #13", 13, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #14", 14, 0, 1, 0, true));
+
+                            t.Add(new Temperature("CPU", 1));
+                            t.Add(new Temperature("MotherBoard", 2));
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs:2388]: Motherboard

File: LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs [hunk 2]
```diff
@@ -2365,6 +2365,91 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
...
+                            //todo
+                            //Find a way to read the value of VRM HS FAN
+                            //It is definitely not controllable as it only supports ON / OFF options but the RPM can be read somewhere
+
+                            for (int i = 0; i < superIO.Fans.Length; i++)
+                            {
+                                switch (i)
+                                {
+                                    case 0:
+                                        f.Add(new Fan("Chassis fan", 0));
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs:2420]: Fan (also below)

File: LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs [hunk 3]
```diff
@@ -2365,6 +2365,91 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
...
+                            //t.Add(new Temperature("#Unused 10", 10));
+                            t.Add(new Temperature("PCH Chip CPU Max", 11));
+                            t.Add(new Temperature("PCH Chip", 12));
+                            t.Add(new Temperature("PCH CPU", 13));
+                            t.Add(new Temperature("PCH MCH", 14));
+                            t.Add(new Temperature("Agent 0 DIMM 0", 15));
+                            //t.Add(new Temperature("Agent 0 DIMM 1", 16));
+                            t.Add(new Temperature("Agent 1 DIMM 0", 17));
+                            //t.Add(new Temperature("Agent 1 DIMM 1", 18));
+                            t.Add(new Temperature("Device 0", 19));
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs:2405]: Any better name?

**Review by PhyxionNL — CHANGES_REQUESTED** [review]
See comments.

File: LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs (hunk 1 above)
Comment (Pomianowski) [reply LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs:2388]: I just wanted to keep the names according to AI Suite for easier understanding for users

File: LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs (hunk 2 above)
Comment (Pomianowski) [reply LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs:2420]: I just wanted to keep the names according to AI Suite for easier understanding for users

File: LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs (hunk 3 above)
Comment (Pomianowski) [reply LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs:2405]: I noticed that others for the B550 board added these devices, but I am unable to verify if my ram or other components are returning values here, so I left it as is.

File: LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs (hunk 2 above)
Comment (PhyxionNL) [reply LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs:2420]: It should be Chassis **F**an (with a capital) to follow naming in app :) 
Same applies to Motherboard.

Comment (Pomianowski) [reply LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs:2420]: image

Comment (PhyxionNL) [reply LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs]: All good now to merge?

Comment (Pomianowski) [reply LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs]: I think so, it looks fine to me
image

**Comment by PhyxionNL** [comment]
Thanks @Pomianowski, merged!

**Pull request merged**